tools/dtest/tests/add_down_node_bring_up.go

```diff
@@ -0,0 +1,81 @@
+package dtests
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/m3db/m3db/tools/dtest/harness"
+	"github.com/m3db/m3db/tools/dtest/util"
+	"github.com/m3db/m3db/x/m3em/convert"
+	m3dbnode "github.com/m3db/m3db/x/m3em/node"
+
+	"github.com/m3db/m3em/node"
+	xclock "github.com/m3db/m3x/clock"
+	"github.com/m3db/m3x/log"
+	"github.com/spf13/cobra"
+)
+
+var (
+	addDownNodeAndBringUpTestCmd = &cobra.Command{
+		Use:   "add_down_node_bring_up",
+		Short: "Run a dtest where a node that is DOWN, is added to the cluster. Node is then brought up.",
+		Long:  "",
+		Example: `
+TODO(prateek): write up`,
+		Run: addDownNodeAndBringUpDTest,
+	}
+)
+
+func addDownNodeAndBringUpDTest(cmd *cobra.Command, args []string) {
+	if err := globalCLIOpts.Validate(); err != nil {
+		printUsage(cmd)
+		return
+	}
+	logger := xlog.NewLogger(os.Stdout)
+	logger.Infof("============== %v  ==============", cmd.Name())
```

Comment:
Should we create a reuseable method that both gives you a logger and prints out the name for consistency among the tests?